.rubocop.yml

```diff
@@ -42,6 +42,8 @@ Metrics/BlockLength:
 RSpec/NestedGroups:
   Enabled: true
   Max: 4
+  Exclude:
```

Comment:
The nesting feels like it can be avoided, could we attempt that rather than excluding the spec here, maybe if we pass in the different courses/locations/providers into the wizard then we can avoid the extra setup/nesting?